distribution/lib/Standard/Table/0.0.0-dev/src/Data/Table_Conversions.enso

```diff
@@ -39,13 +39,55 @@ Text.from (that : Table) (format:Delimited_Format = Delimited_Format.Delimited '
          example_to_table =
              json = Examples.simple_table_json
              headers = Examples.simple_table_json_headers
-             json.to_table headers
-Json.to_table : Vector -> Table
-Json.to_table self fields=Nothing = case self of
-    Json.Array items ->
-        rows = items.map item-> case item of
-            Json.Object row -> fields.map n-> row.get n . unwrap . catch Any (_ -> Nothing)
-            _ -> Vector.fill fields.length Nothing
-        cols = fields.map_with_index i-> n-> [n, rows.map (_.at i)]
-        Table.new cols
-    _ -> Invalid_JSON_Format.throw "Json.to_table expects a JSON array."
+             Table.from_json json headers
+Table.from_json : Any -> Vector | Nothing -> Table
+Table.from_json value fields=Nothing =
```

Comment:
Shouldn't this actually be called something along the lines of `from_js_object`?

Seems that it should be to be consistent - we seem to say that `JSON` is a special form of `Text`, so it should include parsing too.